tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py

```diff
@@ -20,6 +20,7 @@ class TendrlBrickUtilizationPlugin(
 
     def __init__(self):
         self.provisioner_only_plugin = False
+        self.brick_path_repl = ":"
```

Comment:
Why don't we define this as part of configuration? Having inside code still restricts as to change this to something else later would need code changes.

Reply:
But i feel this might hide from the user that why i defined in each source